ocs_ci/ocs/fiojob.py

```diff
@@ -312,7 +312,13 @@ def get_pool_name(fixture_name):
     Return ceph pool name based on fixture name suffix.
     """
     if fixture_name.endswith("rbd"):
-        ceph_pool_name = "ocs-storagecluster-cephblockpool"
+        if config.ENV_DATA["platform"].lower() in constants.MANAGED_SERVICE_PLATFORMS:
```

Comment:
As far as I understand, if you run this code on provider, there will be no cephblockpool with provider cluster's clusterId.

Reply:
Yes. The code was updated to work only with consumer but this is only a partial fix.

Reply:
I think we could just return None if the cluster is Provider.